README.md

```diff
@@ -66,3 +66,11 @@ Run all from the testing tab in VSCode or poetry run pytest test
 copy the files from folder ansible_files/ to your controller node then run
 > ansible-playbook todo-playbook.yml -i ansible-inventory
 note: inventory will be needed to be edited for appropriate slave node IPs
+
+
+###Docker image build
+ docker build --target development --tag todo-app:dev .
+ docker build --target production --tag todo-app:prod .
+
+Dev -   docker run --env-file .env todo-app:dev
+Prd -   docker run --env-file .env todo-app:prod
```

Comment:
This runs the app, but it doesn't actually make it accessible! Both are missing the port mappings to allow us to actually reach the container from our host machine, and the dev one is missing our bind-mount to allow for hot-reloading:
```suggestion
Dev -   docker run --env-file .env -p 5000:5000 --mount type=bind,source="$(pwd)"/todo_app,target=/todo_app todo-app:dev
Prd -   docker run --env-file .env -p 5000:5000 todo-app:prod
```